tests/autorestart_test/test_container_autorestart.py

```diff
@@ -0,0 +1,354 @@
+"""
+Test the auto-restart feature of containers
+"""
+import time
+import pytest
+import logging
+
+from collections import defaultdict
+
+from tests.common.utilities import wait_until
+from tests.common.helpers.assertions import pytest_assert
```

Comment:
Reorganize imports per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports) and sort alphabetically within each group:

>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports.
>You should put a blank line between each group of imports.


```
import logging
import time
from collections import defaultdict

import pytest

from tests.common.utilities import wait_until
from tests.common.helpers.assertions import pytest_assert
```